### DISCUSSION TITLE
Is there a suggest way to negate a number?

### DISCUSSION THREAD

**Discussion by johnhooks** [dinerojs/dinero.js#489] [Q&A] [answered]
Is multiplying by an amount of -1 the typical way to negate a number? This works, but I am curious if I missed something in the documentation.

```js
const d = multiply(dinero({ amount: 1000, currency: USD }), { amount: -1 });
```

**Comment by sarahdayan** [comment, marked as answer]
Hey @johnhooks, multiplying by -1 is indeed the recommended way to negate a Dinero object.

However, you don't have to use a scaled amount for this since -1 is an integer:

```js
const d = multiply(dinero({ amount: 1000, currency: USD }), 1);
```

**Reply by johnhooks** [reply]
Awesome, thank you. Though I think you may have meant this?

```js
const d = multiply(dinero({ amount: 1000, currency: USD }), -1);
```

**Reply by sarahdayan** [reply]
Correct! It's been a long week 😁
